src/common/locales/strings/enUS.json

```diff
@@ -36,7 +36,6 @@
   "change_recovery_question4": "What was your favorite brand of clothing in high school?",
   "change_recovery_question5": "What is the birthdate of your favorite grand parent?",
   "change_recovery_question6": "What is the birthdate of your favorite niece or nephew?",
-  "change_recovery_question7": "What was the first address you remember living in?",
```

Comment:
No need to change this, but FYI you only need to remove the enUS.json string. The crowdin automation will handle the other languages.